surfactant/cmd/generate.py

```diff
@@ -348,13 +357,20 @@ def sbom(
                 parent_entry = None
                 parent_uuid = None
 
+            # Replace entry install prefix with user specified value if given by cli args
+            if install_prefix_:
+                if entry.installPrefix:
+                    logger.warning(
+                        f"Conflicting installPrefix definitions; Overriding config file ({entry.installPrefix}) with CLI argument ({install_prefix_})"
+                    )
+
+                entry.installPrefix = install_prefix_
```

Comment:
I wonder if it would be safer to say that the `--install_prefix` CLI option can only be used when Surfactant is given a file/folder as input rather than a context file? Then we just can make it an error to give an install prefix flag with the context file and exit before generating an SBOM.

If someone has gone through the effort of creating a context file with install prefixes specified for entries, it seems like overriding them with the CLI option would only ever happen as a mistake.